include/aspect/particle/output/null.h

```diff
@@ -0,0 +1,78 @@
+/*
+ Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file doc/COPYING.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef __aspect__particle_output_null_h
+#define __aspect__particle_output_null_h
+
+#include <aspect/particle/output/interface.h>
+
+
+namespace aspect
+{
+  namespace Particle
+  {
+    namespace Output
+    {
+      /**
+       * Blank class to avoid output of data, for example, if particles are used to represent
+       * other physical phenomenon that influences the simulation and we don't care about their positions
```

Comment:
Just remove the class and let the manager check whether anything needs to be called at all. You could also allow a list of output objects that will all be called in term. If that list happens to be empty, well, then that's just how it is. I think that's actually the most elegant solution. It would work in the same way as we do for other postprocessors.